### PR TITLE
Update de.po

### PR DESCRIPTION
**Pull request by Ceeee**
German texts for new features added, various minor corrections of dictionary mistakes

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -256,13 +256,13 @@ msgid "Build a subber"
 msgstr "Baut einen Kettentaucher"
 
 msgid "Build a target bot"
-msgstr ""
+msgstr "Baut einen Zielroboter"
```
Comment (Emxx52) [inline po/de.po:259]: Minor inconsistency: TargetBot is translated as *Mobile Zielscheibe* (object name/SatCom).

File: po/de.po [hunk 2]
```diff
@@ -277,10 +277,10 @@ msgid "Build a tracked sniffer"
 msgstr "Baut einen Kettenschnüffler"
 
 msgid "Build a vault"
-msgstr ""
+msgstr "Baut einen Speicher"
```
Comment (Emxx52) [inline po/de.po:280]: Minor inconsistency: Vault is translated as *Bunker* (object name/SatCom).

File: po/de.po [hunk 3]
```diff
@@ -277,10 +277,10 @@ msgid "Build a tracked sniffer"
...
 
 msgid "Build a wheeled builder"
-msgstr ""
+msgstr "Baut einen Radtransporter"
```
Comment (Emxx52) [inline po/de.po:283]: *Radkonstrukteur*

File: po/de.po [hunk 4]
```diff
@@ -1435,7 +1435,7 @@ msgid "Run research program for shooter"
 msgstr "Forschungsprogramm Shooterkanone"
 
 msgid "Run research program for target bot"
-msgstr ""
+msgstr "Forschungsprogramm Ziel-Roboter"
```
Comment (Emxx52) [inline po/de.po:1438]: Minor inconsistency: TargetBot is translated as *Mobile Zielscheibe* (object name/SatCom).

File: po/de.po [hunk 5]
```diff
@@ -1537,13 +1537,13 @@ msgid "Sound\\Music and game sound volume"
...
-msgstr ""
+msgstr "Weltraumforscher\\Deaktiviert Fähigkeiten des Astronauten"
 
 msgid "Space Programmer\\Disables radio-control"
-msgstr ""
+msgstr "Weltraumprogrammierer\\Deaktiviert Funkfernsteuerung"
 
 msgid "Space Researcher\\Disables using all previously researched technologies"
-msgstr ""
+msgstr "Weltraumwissenschaftler\\"
```
Comment (Emxx52) [inline po/de.po:1546]: Missing translation.

**Review by Emxx52 — CHANGES_REQUESTED** [review]

File: po/de.po (hunk 2 above)
Comment (Ceeee) [reply po/de.po:280]: What use is the structure? Storing data? Then "Speicher" is correct. "Bunker" has the same meaning in German and English: A place to protect people in when (ballistic) attacks happen.

Comment (Emxx52) [reply po/de.po:280]: In that case *Bunker* would indeed be correct, it does not store data.

File: po/de.po (hunk 1 above)
Comment (Ceeee) [reply po/de.po:259]: Adapted.

File: po/de.po (hunk 3 above)
Comment (Ceeee) [reply po/de.po:283]: Adapted.

File: po/de.po (hunk 5 above)
Comment (Ceeee) [reply po/de.po:1546]: Completed.

File: po/de.po (hunk 4 above)
Comment (Ceeee) [reply po/de.po:1438]: Adapted.

File: po/de.po (hunk 2 above)
Comment (Emxx52) [reply po/de.po]: https://colobot.info/SatComOnline/dev/object/safe

Comment (Ceeee) [reply po/de.po]: Alright. Sorry. I mistook that to be part of the signal infrastructure. Integrated and committed, Do I have to do anything to the PR to reflect the changes in that or can you pull the newest commits along?

Comment (Ceeee) [reply po/de.po]: SatCom in the browser is seriously cool!

Comment (Emxx52) [reply po/de.po]: Everything checks out and it's ready to be merged. Thank you!

**Review by Emxx52 — APPROVED** [review]

**Pull request merged**